src/main/java/com/sfuentes/Zoo.java

```diff
@@ -0,0 +1,27 @@
+package com.sfuentes;
+
+public class Zoo {
+
+  public String calculatePercentage(int[] agesAnimals) {
+    double countBetweenZeroAndOne = 0;
+    double countLessThanThree = 0;
+    double countGreaterThanOrEqualToThree = 0;
+
+    for (int ageAnimal : agesAnimals) {
+      if (ageAnimal <= 1) {
+        countBetweenZeroAndOne++;
```

Comment:
replace countBetweenZeroAndOne with betweenZeroAndOneCount.
Do the same for other variable names starting with count.